test/e2e/framework/framework.go

```diff
@@ -259,6 +259,80 @@ func (f *Framework) matchNginxConditions(name string, matcher func(cfg string) b
 	}
 }
 
+// GetNginxConfigMap gets ingress-nginx's nginx-configuration map
+func (f *Framework) GetNginxConfigMap() (*v1.ConfigMap, error) {
+	if f.KubeClientSet == nil {
+		return nil, fmt.Errorf("KubeClientSet not initialized")
+	}
+
+	config, err := f.KubeClientSet.CoreV1().ConfigMaps("ingress-nginx").Get("nginx-configuration", metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	time.Sleep(1 * time.Second)
```

Comment:
why do you need to sleep here?

Reply:
Leftover from building off of the update method! Nice catch, will delete :)